pkg/pillar/types/errortime.go

```diff
@@ -124,7 +248,7 @@ func (etsPtr *ErrorAndTimeWithSource) HasError() bool {
 // ErrorAndTime returns instance of ErrorAndTime corresponding to
 //  Error and ErrorTime in the instance of ErrorAndTimeWithSource
 func (etsPtr *ErrorAndTimeWithSource) ErrorAndTime() ErrorAndTime {
```

Comment:
Do we use thins function? Seems like the useful function would also will in a source as part of creating an ErrorAndTimeWithSource.

Reply:
I refactored caller to use ErrorDescription

Reply:
Do you have a func (edPtr *ErrorDescription) SetErrorDesc() which can do the common part of setting time to Now if not set, set severity to error if unspecified? Basically getthing that code in one and only one place.

Reply:
Done

Reply:
See below.